service.py

```diff
@@ -317,7 +321,7 @@ def get_subject(subjectID):
         hot_article.append(x)
 
     return render_template('subject.html', url=url, subject_id=subject.id, articles=articles, hot_article=hot_article, Tool=Tool)
-
+    return render_template('io.html', url=url, subject_id=subject.id, articles=articles, hot_article=hot_article, Tool=Tool)
```

Comment:
@CZH-Gaizka 

Will this return statement  (line 324) be execute, since there is another return statement just above this one (on line 323).  -Hui